tron/mesos.py

```diff
@@ -309,13 +311,16 @@ def handle_next_event(self, deferred_result=None):
         self.deferred.addErrback(logError)
         self.deferred.addErrback(self.handle_next_event)
 
-    def submit(self, task):
+    def submit(self, task, recover=False):
         if not task:
             return
 
         if not self.enabled:
-            task.log.info('Task failed to start, Mesos is disabled.')
-            task.exited(1)
+            if recover:
+                task.log.info('Could not recover task, Mesos is disabled.')
```

Comment:
I guess we don't need to belabor the point by logging lots of advice, but can you confirm that we will be "unknown" in this case? (we should)
If we are disabling mesos for some reason, and there is a taskid, then it might be still running but who knows.

It just says "return" on line 324, does that mean it won't move the state machine at all and leave it `unknown`?

Reply:
Good catch! Yes, the return means that the state machine isn't moved, and it was my intention for it be be unknown. But since it's transitioned to running in `tron/core/actionrun`, there's a chance it could be left in running. Will update.